build/buildall

```diff
@@ -57,6 +57,13 @@ function bloopInstall() {
 
   [[ "$BUILD_ALL_DEBUG" == "1" ]] && set -x
 
+  local javaVersion=$(java -version 2>&1 | sed -E -n 's/.* version "([^.-]*).*"/\1/p' | cut -d' ' -f1)
+  if [ $javaVersion -lt 11 ]
+  then
+    echo "Bloop install requires JDK >= 11"
+    exit 1
+  fi
```

Comment:
It might be tricky but ideally we need to put this requirement into the pom as an enforcer rule so the direct maven users are covered as well.

Reply:
I don't see how the enforcer can apply a rule when a user explicitly specifies `ch.epfl.scala:bloop-maven-plugin:bloopInstall` at the command line. Even with an enforcer rule that always fails, it seems the enforcer rule is not applied when the plugin target is explicitly provided by the user.

Reply:
I filed #9621. I will push a branch based on your branch  with a suggestion what it might look like

Reply:
Ah, I see that perhaps using a `default-cli` execution configuration will allow interaction with the command line invocation.

Reply:
Something like this, conditionally activated via a profile _should_ work:
```
            <plugin>
                <groupId>ch.epfl.scala</groupId>
                <artifactId>bloop-maven-plugin</artifactId>
                <executions>
                    <execution>
                        <id>default-cli</id>
                        <configuration>
                            <skip>true</skip>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

But, it looks to me like the the `skip` configuration in the extension is [not properly handled](https://github.com/scalacenter/bloop-maven-plugin/blob/2a0604da347e89ecf09170837c0f4738e6e0d780/src/main/java/bloop/integrations/maven/BloopMojo.java#L80C21-L80C25). They should be checking it during execute and ending early if present, similar to [this other plugin](https://github.com/apache/maven-assembly-plugin/blob/d1efcdaee4fc1a448deeb93c13f8f8fdd1428fcb/src/main/java/org/apache/maven/plugins/assembly/mojos/AbstractAssemblyMojo.java#L454).

Reply:
We can  put the definition of bloopInstall plugin into a dedicated profile and then we don't have to worry about its skip implementation. Take a look into my PR into your PR branch PhilDakin/spark-rapids#1